include/model_mgmt.h

```diff
@@ -64,6 +64,7 @@ namespace reinforcement_learning { namespace model_management {
     enum class model_type_t {
       UNKNOWN,
       CB,
+      CA,
```

Comment:
Enum entries must always be append otherwise it's a breaking change.

Reply:
Changed to be appended